tests/lowlevel/test_width.py

```diff
@@ -0,0 +1,69 @@
+from msdsl.expr.format import SIntFormat, UIntFormat
```

Comment:
This is a new test that looks at how widths are calculated for integer constants.